rpyscope/gui_qt.py

```diff
@@ -509,35 +603,39 @@ def set_path(self):
             self.path_input.setText(str(path))
 
     def reset_resolution(self):
-        self.res_input.setText(self.config.get("default_resolution"))
+        self.res_input.setText(self.config._get_default("resolution"))
 
     def set_resolution(self):
         new_res = self.res_input.text()
-        if self.cam.resolution != new_res:
+        if str(self.cam.resolution) != new_res:
```

Comment:
Why turn this into a `str`, shouldn't it always be a string?

Reply:
Before I turned it into `str`, they were never equal. This is because `type(self.cam.resolution)` returns `picamera.mmalobj.PiResolution`